credsweeper/scanner/scan_type/scan_type.py

```diff
@@ -154,11 +154,16 @@ def _get_candidate(cls, config: Config, rule: Rule, target: AnalysisTarget) -> O
             remove current line. None otherwise
 
         """
+        if target.line.strip() in config.exclude_lines:
```

Comment:
then, here may be extra work when user suggests skip values only

Reply:
Ok, agree, will update

Reply:
@babenek fixed with 3670e584232a25ef4b2a561e381ac184d1b6057a